test/unit/shadow/polyfill.js

```diff
@@ -81,6 +81,13 @@ describe('shadow/polyfill', function () {
     it('innerHTML', function () {
       root.innerHTML = '<slot name="test"></slot>';
       expect(root.childNodes[0].name).to.equal('test');
+
+      if (canPatchNativeAccessors) {
```

Comment:
Maybe I'm not understanding something – why can't we run this test no matter what `canPatchNativeAccessors` is?